scripts/download_backtest_data.py

```diff
@@ -8,23 +8,25 @@
 
 from freqtrade import (exchange, arguments, misc)
 
-DEFAULT_DL_PATH = 'freqtrade/tests/testdata'
+DEFAULT_DL_PATH = 'user_data/data'
 
 arguments = arguments.Arguments(sys.argv[1:], 'download utility')
 arguments.testdata_dl_options()
 args = arguments.parse_args()
 
 TICKER_INTERVALS = ['1m', '5m']
-PAIRS = []
 
-if args.pairs_file:
-    with open(args.pairs_file) as file:
-        PAIRS = json.load(file)
-PAIRS = list(set(PAIRS))
+dl_path = args.export if args.export and os.path.exists(args.export) \
```

Comment:
If you need to break your oneliner to multiple lines then I'm pretty sure doing the old-fasioned way
```
if..
else..
```
might be a better idea

Reply:
Done.